docs/fedora-releases-guide.md

```diff
@@ -56,11 +64,15 @@ Here are the key differences between the two:
 <tr>
 <th>Packit CLI</th>
 <td><p>Have a <code>.packit.yaml</code> in <b>upstream</b> repo, clone repo and run:</p>
-  <pre class="language-yaml">packit propose-downstream</pre>
+
+  <pre>packit propose-downstream</pre></br>
```

Comment:
I don't think the `</br>` should be there. Also, it should be `<br/>`, not `</br>`.